### PR TITLE
181933 storage container structure changes

### PR DESCRIPTION
**Pull request by pravinvarade10801**
Implementation of event wise storage container structure.

### CODE REVIEW

File: src/UKHO.ERPFacade.CleanUp.WebJob/Services/CleanUpService.cs [hunk 1]
```diff
@@ -25,25 +26,30 @@ public CleanUpService(ILogger<CleanUpService> logger,
...
+                TimeSpan timediff = DateTime.Now - Convert.ToDateTime(entity["RequestDateTime"].ToString());
+
                 if (timediff.Days > int.Parse(_erpFacadeWebjobConfig.Value.CleanUpDurationInDays))
                 {
-                    Task.FromResult(_azureTableReaderWriter.DeleteEESEntity(entity.CorrelationId));
-                    _logger.LogInformation(EventIds.DeletedContainerSuccessful.ToEventId(), "Deleting container : {0}", entity.CorrelationId);
-                    _azureBlobEventWriter.DeleteContainer(entity.CorrelationId.ToLower());
+                    Task.FromResult(_azureTableReaderWriter.DeleteEntity(entity["CorrelationId"].ToString(), tableName));
+
+                    _logger.LogInformation(EventIds.DeletedContainerSuccessful.ToEventId(), "Deleting directory {0} from {1} container", entity["CorrelationId"].ToString(), eventContainerName);
```
Comment (Vishal14583) [inline src/UKHO.ERPFacade.CleanUp.WebJob/Services/CleanUpService.cs:50]: Can we use variable names in string interpolation to keep it consistent with other logs.

File: src/UKHO.ERPFacade.CleanUp.WebJob/UKHO.ERPFacade.CleanUp.WebJob.csproj [hunk 2]
```diff
@@ -51,7 +51,7 @@
 
   <ItemGroup>
     <None Update="appsettings.local.overrides.json">
-      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (Vishal14583) [inline src/UKHO.ERPFacade.CleanUp.WebJob/UKHO.ERPFacade.CleanUp.WebJob.csproj:54]: This is not required as it is override file.

File: src/UKHO.ERPFacade.CleanUp.WebJob/Services/CleanUpService.cs [hunk 3]
```diff
@@ -25,25 +26,30 @@ public CleanUpService(ILogger<CleanUpService> logger,
...
+            CleanUpEvents(Constants.S57EventTableName, Constants.S57EventContainerName);
         }
 
-        //Private methods       
-        private void CleanUpBlobsAndTablesRelatedToEESEvent()
+        private void CleanUpEvents(string tableName, string eventContainerName)
         {
-            _logger.LogInformation(EventIds.FetchEESEntities.ToEventId(), "Fetching all EES entities from azure table");
-            var entities = _azureTableReaderWriter.GetAllEntityForEESTable();
+            _logger.LogInformation(EventIds.FetchEESEntities.ToEventId(), "Fetching all records from azure table {0}", tableName);
```
Comment (Vishal14583) [inline src/UKHO.ERPFacade.CleanUp.WebJob/Services/CleanUpService.cs:35]: Can we use variable names in string interpolation to keep it consistent with other logs.

**Review by Vishal14583 — CHANGES_REQUESTED** [review]
Few review comments

**Review by rushikeshkhopatkar — APPROVED** [review]
Looks good.

**Review by rushdynajath — APPROVED** [review]
looks good

**Pull request merged**